ravenframework/utils/randomUtils.py

```diff
@@ -87,6 +88,7 @@ def testSampling(self, n=1e5,engine=None):
 if stochasticEnv == 'numpy':
   npStochEnv = np.random.RandomState()
 else:
+  setupCpp()
```

Comment:
Hm, can this by put in some kind of if statement or try statement, because right now setupCpp unconditionally adds things to the python path?

Reply:
(Or modify setupCpp to check before adding paths)

Reply:
Okay I added a check in `setupCpp()`